src/main/java/io/meeds/qa/ui/pages/page/factory/space/SpaceHomePage.java

```diff
@@ -1215,4 +1214,40 @@ public void viewAllRepliesInCommentsDrawer(String comment) {
     getCommentsDrawerViewAllReplies(comment).clickOnElement();
   }
 
+  private BaseElementFacade getReactionActivityList(String activity) {
```

Comment:
Please reorder methods to make `public` methods on top of the class, before the `private` methods.
In general case, the `private` methods must go on bottom of the class.

Reply:
methods reordered.